libkbfs/md_ops_test.go

```diff
@@ -210,7 +211,8 @@ func putMDForPrivate(config *ConfigMock, rmd *RootMetadata) {
 		EncryptedPrivateMetadata{}, nil)
 	config.mockBsplit.EXPECT().ShouldEmbedBlockChanges(gomock.Any()).
 		Return(true)
-	config.mockMdserv.EXPECT().Put(gomock.Any(), gomock.Any(), gomock.Any()).Return(nil)
+	config.mockMdserv.EXPECT().Put(gomock.Any(), gomock.Any(), gomock.Any(),
+		gomock.Any(), gomock.Any()).Return(nil)
```

Comment:
Since you're passing `nil` everywhere, shouldn't the mock server be expecting a `nil`?  That said, it would be nice to make sure that the right non-`nil` thing gets plumbed through to the server. but I'm ok with doing that as part of a higher-level test if you want.

Reply:
I was worried if we were gonna change some stuff in tests that break these mock tests. But I guess we can set them all to be `nil` and fix the broken ones when we start using locks.